### PR TITLE
Remove sed from plugin registry Dockerfile sync

### PR DESCRIPTION
**Pull request by ericwill**
Part of CRW-1831

Signed-off-by: Eric Williams <ericwill@redhat.com>

### CODE REVIEW

File: codeready-workspaces-pluginregistry/build/scripts/sync.sh [hunk 1]
```diff
@@ -81,10 +81,6 @@ popd >/dev/null || exit
 
 # transform Dockefile
 sed "${TARGETDIR}/build/dockerfiles/Dockerfile" --regexp-extended \
-    `# Replace image used for registry with rhel8/httpd-24` \
-    -e 's|^ *(FROM.*rhscl/httpd.*)|#\1|' \
-    -e 's|^ *FROM registry.access.redhat.com/.* AS registry|# &|' \
-    -e 's|# *(FROM.*rhel8/httpd.*)|\1|' \
```
[the remark this answers is not in the document]
Comment (nickboldt) [reply codeready-workspaces-pluginregistry/build/scripts/sync.sh:87]: no, rhel8.4 image doesn't exist internally at OSBS container registry. Only the new ubi8 8.4 one does.

[the remark this answers is not in the document]
Comment (nickboldt) [reply codeready-workspaces-pluginregistry/build/scripts/sync.sh:87]: no, rhel8.4 image doesn't exist internally at OSBS container registry. Only the new ubi8 8.4 one does.

[the remark this answers is not in the document]
Comment (nickboldt) [reply codeready-workspaces-pluginregistry/build/scripts/sync.sh:87]: no, rhel8.4 image doesn't exist internally at OSBS container registry. Only the new ubi8 8.4 one does.

**Review by nickboldt — APPROVED** [review]
+1, but also need to do the same for dashboard and devfile reg.

File: codeready-workspaces-pluginregistry/build/scripts/sync.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (benoitf) [reply codeready-workspaces-pluginregistry/build/scripts/sync.sh:87]: ok copy that 3 times ^^

**Pull request merged**